google-cloud-pubsublite/src/main/java/com/google/cloud/pubsublite/internal/wire/ConnectedSubscriberImpl.java

```diff
@@ -122,23 +88,18 @@ protected void handleStreamResponse(SubscribeResponse response) throws CheckedAp
         onMessages(response.getMessages());
         return;
       case SEEK:
-        onSeekResponse(response.getSeek());
-        return;
+        throw new CheckedApiException(
+            String.format(
+                "Received seek response from client which never sends seek requests %s.",
+                initialRequest),
+            Code.FAILED_PRECONDITION);
       default:
         throw new CheckedApiException(
             "Received a message on the stream with no case set.", Code.FAILED_PRECONDITION);
     }
   }
 
   private void onMessages(MessageResponse response) throws CheckedApiException {
-    try (CloseableMonitor.Hold h = monitor.enter()) {
-      if (seekInFlight) {
-        log.atInfo().log(
-            "Dropping %s messages due to outstanding seek on stream with initial request %s.",
-            response.getMessagesCount(), initialRequest);
-        return;
-      }
-    }
     checkState(
         response.getMessagesCount() > 0,
         String.format(
```

Comment:
Does `PullResponse` need to be updated, or deliberately used in the error message?

Reply:
Done.